File: Modul 2 Sem 1/Phython/PRAK205-2410817110005-MuhammadRizqanKarim.py
import math

#test case 1
A1, B1 = map(int, input(" ").split())

C1 = math.sqrt((B1 ** 2) - (A1 ** 2))
Keliling1 = A1 + B1 + C1
Luas1 = 0.5 * C1 * A1

print(" ")
print(f"Alas = {C1:.0f} cm")
print(f"Tinggi = {A1:.0f} cm")
print(f"Keliling = {Keliling1:.0f} cm")
print(f"Luas = {Luas1:.0f} cm^2")

#test case 2
A = int(input(" "))
B = int(input(" "))

C = math.sqrt((B ** 2) - (A ** 2))
Keliling = A + B + C
Luas = 0.5 * C * A

print(" ")
print(f"Alas = {C:.0f} cm")
print(f"Tinggi = {A:.0f} cm")
print(f"Keliling = {Keliling:.0f} cm")
print(f"Luas = {Luas:.0f} cm^2")

File: Modul 2 Sem 1/C/PRAK205-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>
#include <math.h>

int main()
{
    //test case 1
    int A1, B1, C1, Keliling1, Luas1;

    scanf("%d %d", &A1, &B1);

    C1 = sqrt((B1 * B1) - (A1 * A1));
    Keliling1 = A1 + B1 + C1;
    Luas1 = 0.5 * C1 * A1;

    printf("\nAlas = %d cm\n", C1);
    printf("Tinggi = %d cm\n", A1);
    printf("Keliling = %d cm\n", Keliling1);
    printf("Luas = %d cm^2\n", Luas1);

    //test case 2
    int A, B, C, Keliling, Luas;

    scanf("%d", &A);
    scanf("%d", &B);

    C = sqrt((B * B) - (A * A));
    Keliling = A + B + C;
    Luas = 0.5 * C * A;

    printf("\nAlas = %d cm\n", C);
    printf("Tinggi = %d cm\n", A);
    printf("Keliling = %d cm\n", Keliling);
    printf("Luas = %d cm^2\n", Luas);

    return 0;
}
